newbank/server/NewBankClientHandler.java

```diff
@@ -72,7 +75,27 @@ public void run() {
 		}
 	}
 
-	public CustomerID login() throws IOException {
+	public CustomerID startUp() throws IOException {
+		boolean startUp = true;
+		try {
+			while (startUp) {
+				out.println("Login or Setup New Customer?");
+				String startUpString = in.readLine();
+				if (startUpString.equals(Constants.startLogin)){
+					startUp = false;
+					break;
+				}
+				if (startUpString.equals(Constants.startSetupNewCustomer)){
+					out.println("Enter Customer Name");
+					String customerName = in.readLine();
+					Customer newCustomer = theCustomerManager.createCustomer(customerName);
```

Comment:
This allows us to have multiple customers with the same name. Usernames should be unique, otherwise I could be trying to login to my account but keep landing in someone else's

Reply:
Agreed, I missed that!